src/euphorie/client/tests/test_risk.py

```diff
@@ -233,5 +234,6 @@ def test_download(self):
                     {
                         "accept-ranges": "bytes",
                         "content-type": "image/png",
+                        'link': '<http://nohost/plone/client/nl/ict/software-development/%2B%2Bsession%2B%2B1/1/@@download/image_training/dummy.gif>; rel="canonical"',  # noqa: E501
```

Comment:
I saw pre-commit make a change locally, fixing the single quotes of link:

```suggestion
                        "link": '<http://nohost/plone/client/nl/ict/software-development/%2B%2Bsession%2B%2B1/1/@@download/image_training/dummy.gif>; rel="canonical"',  # noqa: E501
```

Reply:
Thanks again. I updated pre-commit to make it pass